ably/realtime_channel_integration_test.go

```diff
@@ -93,6 +93,82 @@ func TestRealtimeChannel_Subscribe(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestRealtimeChannel_SubscriptionFilters(t *testing.T) {
+	app, err := ablytest.NewSandbox(nil)
+	assert.NoError(t, err)
+	defer app.Close()
+	options := app.Options()
+	restClient, err := ably.NewREST(options...)
+	assert.NoError(t, err)
+	msg := []*ably.Message{
+		{
+			Name: "filtered",
+			Data: "This should not be filtered",
+			Extras: map[string]interface{}{
+				"headers": map[string]interface{}{
+					"name":   "value one",
+					"number": 1234,
+					"bool":   true,
+				},
+			},
+		},
+		{
+			Name: "filtered",
+			Data: "filtered messages",
+		},
+		{
+			Name: "filtered",
+			Data: "Another unfiltered message",
```

Comment:
This message will be filtered, since header.number != 1234

Reply:
Yeah, wrong use of filtered term